controllers/runner_controller.go

```diff
@@ -285,8 +290,8 @@ func (r *RunnerReconciler) unregisterRunner(ctx context.Context, repo, name stri
 	return true, nil
 }
 
-func (r *RunnerReconciler) listRunners(ctx context.Context, repo string) ([]GitHubRunner, error) {
-	runners := []GitHubRunner{}
+func (r *RunnerReconciler) listRunners(ctx context.Context, repo string) (GitHubRunnerList, error) {
+	runners := GitHubRunnerList{}
```

Comment:
Could you tell me why you changed to Struct?

Reply:
```
{
  "total_count": 2,
  "runners": [
    {
      "id": 23,
      "name": "MBP",
      "os": "macos",
      "status": "online"
    },
    {
      "id": 24,
      "name": "iMac",
      "os": "macos",
      "status": "offline"
    }
  ]
} 
```
Github had changed API,  as result controller can't unregister the runner

https://github.com/summerwind/actions-runner-controller/issues/11, due to

```2020-03-18T18:34:44.737Z	ERROR	controller-runtime.controller	Reconciler error {"controller": "runner", "request": "actions-runner-system/my-repo-runner", "error": "json: cannot unmarshal object into Go value of type []controllers.GitHubRunner"}```

Reply:
That's good to know. Thank you!